cli/.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": [["@babel/preset-env", { "targets": { "node": 10 }}]]
```

Comment:
👍 
But also let me add running this command from test environments

Reply:
Added https://github.com/cypress-io/cypress-test-node-versions/pull/53

Reply:
thank you